### ISSUE TITLE
Replace uppercase text-transform by capitalized text source, to avoid extraneous accents

### ISSUE THREAD

**Issue by lstamellos** [Chocobozzz/PeerTube#2085]
I think there's an easy way to remove accents from capitalized Greek words. 
I did find this: https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript but I'm no dev so I can't figure out a way to incorporate it.

**Comment by Chocobozzz** [comment]
I don't know Greek, so could you explain why we need to remove accents from capitalized Greek letters?

**Comment by lstamellos** [comment]
Oops, you are right, sorry! In Greek, accents are not put on words written with all capital letters (same goes with small caps too).

**Comment by Chocobozzz** [comment]
Could you update the Greek translations?

**Comment by lstamellos** [comment]
Changing the translation text strings to uppercase would not be the right fix, since that would prevent proper CSS formatting (by the themes or by custom CSS). It's an issue common to many other languages, not only Greek, that need the accents removed when the text is tranformed to uppercase.

**Comment by rigelk** [comment]
TLDR': I think the problem is a narrow use case (user/theme style modification), for a too big perf drop.

Maybe I misunderstood the use case, but I can't see how we do could detect that a theme modified the CSS to make an element uppercase, to then apply that fix. I mean, there is no native event per element for style modification, and asking the DOM to give the computed style can only be polled. Without a narrower problem definition, we are bound to poll every textual element, to check if we should remove accents. That is unpractical.

**Comment by lstamellos** [comment]
@rigelk You are right of course, this is something that need to be implemented on the default theme, not the core itself.

**Comment by rigelk** [comment]
@lstamellos the default them won't have this problem, only third-party themes, in which case it is their problem.

**Comment by lstamellos** [comment]
@rigelk Currently, the default theme *does* have this problem...

**Comment by rigelk** [comment]
@lstamellos I had misumderstood you, then. It means the current theme does CSS transformations with uppercase rules when we should do that directly in the translation source.
